Material ui 0.16

oyvindberg opened the pull request:
todo:
- did i read they did something to styles?
- detection of component methods is apparently broken
- figure out types
```diff
+      case ("AutoComplete",     "animation")                 => "js.Function"
+      case ("AutoComplete",     "onClose")                   => "Callback"
+      case ("DropDownMenu",     "animation")                 => "js.Function"
+      case ("DropDownMenu",     "onClose")                   => "Callback"
+      case ("IconMenu",         "animation")                 => "js.Function"
+      case ("MenuItem",         "animation")                 => "js.Function"
```
```diff
+      case ("AutoComplete"    , "popoverProps",    "object")        => Normal("js.Any")
+      case ("GridList"        , "cellHeight",      "auto")          => Normal("js.Any")
+      case ("RadioButtonGroup", "defaultSelected", "any")           => Normal("js.Any")
+      case ("RadioButtonGroup", "valueSelected",   "any")           => Normal("js.Any")
+      case ("Stepper"         , "children",        "arrayOf(node)") => Normal("js.Any")
```

oyvindberg committed:
Material-ui 0.16.4

timothyklim commented:
Wow! Can't wait when this will be released. 👍

chandu0101 commented:
i added React `createElement` method https://github.com/chandu0101/scalajs-react-components/blob/master/core/src/main/scala/chandu0101/scalajs/react/components/package.scala#L60 

may be you don't need single child checks anymore :)

timothyklim commented:
Any news?

oyvindberg committed:
Material-ui 0.17.0

- Tweaked formatting
- Update callback types from definitely typed
- Fixed member method detection
- Updated demos
- Update deps and docs
